css/common.css

```diff
@@ -98,6 +98,10 @@ summary > .oo-ui-labelElement-label {
 
 /* Wiki list */
 
+.oo-ui-messageWidget.oo-ui-messageWidget-block.oo-ui-flaggedElement-warning {
```

Comment:
Add a new class to the widget using `'classes' => [ 'closedWikiWarning' ]`, then just use that class as the selector here: `.closedWikiWarning {`

Reply:
See the previous comment.